src/doc/reference.md

```diff
@@ -2959,10 +2956,10 @@ meaning of the operators on standard types is given here.
   : Exclusive or.
     Calls the `bitxor` method of the `std::ops::BitXor` trait.
 * `<<`
-  : Logical left shift.
+  : Aritmetic left shift.
```

Comment:
well maybe that documentation was written by someone thinking of languages where `<<` and `>>` always *do* denote purely logical shifts.  (The `<<` operator is indeed a logical shift; its just not a terribly interesting thing to point out, since its trivial in that case.)